lte/gateway/c/core/oai/tasks/grpc_service/amf_service_handler.c

```diff
@@ -36,7 +36,6 @@ status_code_e send_n11_create_pdu_session_resp_itti(
   message_p->ittiMsg.n11_create_pdu_session_response = *itti_msg;
   return send_msg_to_task(&grpc_service_task_zmq_ctx, TASK_AMF_APP, message_p);
 }
-
```

Comment:
Please keep the new line between function definitions.

Reply:
addressed review comment